next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  typescript: {
+    ignoreBuildErrors: true,
```

Comment:
Can we change this to `false` and make sure there are no build errors? I'm doing the same right now with the Cadastre.